packages/ui-components/src/stories/styling/semantic-colors/SemanticColors.vue

```diff
@@ -132,6 +132,8 @@ markClassesUsed([
   'bg-danger-darker',
   'bg-outline-1',
   'bg-outline-2',
-  'bg-outline-3'
+  'bg-outline-3',
```

Comment:
What about those bg-highlight utilities, shouldn't those appear somewhere in the docs?

Reply:
There is a plan to do a follow up branch just around storybook, and getting it up to date with the new design system. This will be done right after this is merged.